### PR TITLE
Add some settings for deepseek-v2-lite-chat

### PR DESCRIPTION
**Pull request by aobp**
## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://sgl-fru7574.slack.com/archives/C09784E3EN6 to discuss your PR.

### CODE REVIEW

**Comment by sleepcoo** [comment]
Have you tried training? I think the problem gemini said does exist.

File: configs/deepseek-v2-lite-eagle3.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "attention_bias": false,
+  "attention_dropout": 0.0,
+  "bos_token_id": 151643,
+  "eos_token_id": 151645,
+  "head_dim": 128,
+  "hidden_act": "silu",
+  "hidden_size": 2048,
+  "initializer_range": 0.02,
+  "intermediate_size": 25600,
```
[the remark this answers is not in the document]
Comment (aobp) [reply configs/deepseek-v2-lite-eagle3.json]: Done

File: configs/deepseek-v2-lite-eagle3.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "architectures": [
+    "LlamaForCausalLMEagle3"
+  ],
+  "attention_bias": false,
+  "attention_dropout": 0.0,
+  "bos_token_id": 151643,
+  "eos_token_id": 151645,
```
[the remark this answers is not in the document]
Comment (aobp) [reply configs/deepseek-v2-lite-eagle3.json]: Done

File: specforge/data/template.py [hunk 3]
```diff
@@ -113,3 +113,13 @@ def get_all_template_names(self) -> List[str]:
...
+
+TEMPLATE_REGISTRY.register(
+    name="deepseek",
+    template=ChatTemplate(
+        assistant_header="Assistant:",
+        user_header="User:",
+        system_prompt="You are a helpful assistant.",
+        end_of_turn_token="",
+    ),
+)
```
[the remark this answers is not in the document]
Comment (aobp) [reply specforge/data/template.py:125]: For deepseek-v2-lite-chat, end_of_turn_token is None; ssistant_header and user_header do not end with '\n'

**Comment by aobp** [comment]
> Have you tried training? I think the problem gemini said does exist.

We have trained deepseek-v2-lite-chat for 10 epochs on ultrachat with the update files. And we tested it on alpaca. The results are as follows:
Evaluating: 100%|
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 0,  Acc: 0.63
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 1,  Acc: 0.60
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 2,  Acc: 0.58
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 3,  Acc: 0.57
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 4,  Acc: 0.56
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 5,  Acc: 0.54
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 6,  Acc: 0.52
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 0, pLoss: 1.02
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 1, pLoss: 1.08
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 2, pLoss: 1.12
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 3, pLoss: 1.15
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 4, pLoss: 1.17
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 5, pLoss: 1.20
Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 6, pLoss: 1.23

Hope this is useful for your question.

**Review by sleepcoo — APPROVED** [review]

**Comment by dongyibo** [comment]
> > Have you tried training? I think the problem gemini said does exist.
> 
> We have trained deepseek-v2-lite-chat for 10 epochs on ultrachat with the update files. And we tested it on alpaca. The results are as follows: Evaluating: 100%| Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 0, Acc: 0.63 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 1, Acc: 0.60 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 2, Acc: 0.58 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 3, Acc: 0.57 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 4, Acc: 0.56 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 5, Acc: 0.54 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 6, Acc: 0.52 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 0, pLoss: 1.02 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 1, pLoss: 1.08 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 2, pLoss: 1.12 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 3, pLoss: 1.15 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 4, pLoss: 1.17 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 5, pLoss: 1.20 Eval in ./train/eagle/data/online_data/alpaca/online_alpaca.json, position 6, pLoss: 1.23
> 
> Hope this is useful for your question.

@aobp May I ask how the accuracy is on the training set?
